Content.Server/Body/BodyCommands.cs

```diff
@@ -144,4 +149,68 @@ public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
             shell.SendText(player, $"No mechanism was found with name {mechanismName}.");
         }
     }
+
+    class HurtCommand : IClientCommand
+    {
+        public string Command => "hurt";
+        public string Description => "Ouch";
+        public string Help => $"Usage: {Command} <type> <amount> (<entity uid/_>) (<ignoreResistance>)";
+
+        public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
+        {
+            // Check if we have enough for the dmg types to show
+            if (args.Length < 1)
+            {
+                shell.SendText(player, Help);
+                return;
+            }
+
+            // Send all damage types if we can't parse (e.g. hurt ?)
+            if (!Enum.TryParse<DamageClass>(args[0], true, out var type))
+            {
+                shell.SendText(player, $"Damage Types:\n{string.Join('\n', Enum.GetNames(typeof(DamageClass)))}");
+                return;
+            }
+
+            var ignoreResistance = false;
+            var entityUid = player.AttachedEntityUid.HasValue ? player.AttachedEntityUid.Value : EntityUid.Invalid;
+            if (args.Length < 2 || !int.TryParse(args[1], out var amount) ||
+                args.Length >= 3 && args[2] != "_" && !EntityUid.TryParse(args[2], out entityUid) || 
+                args.Length >= 4 && !bool.TryParse(args[3], out ignoreResistance))
+            {
+                shell.SendText(player, Help);
+                return;
+            }
+
+            if (player == null)
+            {
+                shell.SendText(player, "Only a player can run this command.");
+                return;
+            }
+
+            if (entityUid == EntityUid.Invalid)
+            {
+                shell.SendText(player, "Not a valid entity.");
+                return;
+            }
+
+            var ent = IoCManager.Resolve<IEntityManager>().GetEntity(entityUid);
```

Comment:
This will throw if the entity doesn't exist, despite what the doc for IEntityManager says. Use TryGetEntity instead.